docs/README.md

```diff
@@ -109,20 +109,32 @@ Redshift.
 [sync_finish_job]: /app/jobs/finish_sync_job.rb
 [data_warehouse]: /app/models/data_warehouse.rb
 
-## Resolving Organizations
+## Resolving Organizations with Sources and Ranking
 
-Yes Bearden imports and exports, but the thing that makes it special is the way
-that it models data and resolves `Organization` records. Rather than simply
-having an email column on the organizations table, for example, we have a table
-of email addresses that point to their organization. This means the actual
-`Organization` has very little on it and serves mostly as a way to relate all
-the details of an entity. Each of these details points back to the `Source` that
-created it, which means that we always know which details are most accurate.
+Yes, Bearden imports and exports, but the thing that makes it special is the way
+it models data and resolves `Organization` records.
 
-Evaluating this accuracy happens at export time, during the process of syncing
-with Redshift. Waiting until this point in the process means means that we
-delay the flattening to as late as possible and reflect the most up-to-date
-information as possible.
+Rather than simply having, for example, an email column on the organizations
+table, we have a table of email addresses that point to their organization. This
+means the actual `Organization` has very little data. It serves mostly as a way
+to connect all the details of an entity.
+
+Each of these details, such as an email, location, phone number and so on,
+points back to the `Source` that created it. Consider the following graph.
+
+![Organization and email graph][org_email_graph]
+
+Given multiple fields, an organization resolves to the highest ranked source for
+that field. Since `email_2` originates from the higher-ranked `source_b`, the
+output to Redshift will only contain that `email_2`. The lesser-ranked values
+are ignored.
```

Comment:
Awesome graph - great idea! My only thought is keeping language consistent regarding higher and lower ranks. In the section about the rules, I wrote that lower rank is better, so maybe flip the language here?